### ISSUE TITLE
Allow setting a gruntfile and other passed in parameters

### ISSUE THREAD

**Issue by aaronbarker** [vladikoff/grunt-devtools#18]
I have a project where I use multiple gruntfile files to keep things organized. Similarly I pass in variables to be used within the grunt file. I love the idea of having everything in Chrome instead of going back to the command line, but need to be able to pass this additional info in.

**Comment by vladikoff** [comment]
Thanks for the issue, I will look into this :)

**Comment by moshner** [comment]
I agree this this request. We are building several gruntfiles in the same folder to take advantage of shared libraries. So we have the following files that I'd love to load from your plugin: Gruntfile.js, Gruntfile-1.js, Gruntfile-2.js, etc...

**Comment by dzuluaga** [comment]
+1. Something like this would do the trick when starting grunt-devtools:

```
grunt-devtools --env=test
```

BTW, the plugin looks really cool! It's a selling point while moving from Maven to Grunt.
